code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,983 @@
+	var/turf/T = get_turf(src)
+	var/list/possible_targets = list()
+	for(var/mob/living/M in T.contents)
+		if(!iscultist(M))
+			possible_targets.Add(M)
+	var/mob/offering
+	if(possible_targets.len > 1)
+		offering = input(user, "Choose an offering to sacrifice.", "Unholy Tribute") as null|anything in possible_targets
 	else
-		usr.whisper("Ta'gh fara[pick("'","`")]qha fel d'amar det!")
-	playsound(U, 'sound/items/Welder2.ogg', 25, 1)
-	var/turf/T = get_turf(U)
-	if(T)
-		T.hotspot_expose(700,125)
-	var/rune = src // detaching the proc - in theory
-	empulse(U, (range_red - 2), range_red)
-	qdel(rune)
-
-///Blood Drain: Deals 1-25 damage to someone on top of it and heals an equivalent amount of brute damage to the user. Can be stacked. Too much blood drain = blood hunger that must be sated.
-
-/obj/effect/rune/proc/drain()
-	var/drain = 0
-	for(var/obj/effect/rune/R in world)
-		if(R.word1==wordtravel && R.word2==wordblood && R.word3==wordself)
-			for(var/mob/living/carbon/D in R.loc)
-				if(D.stat!=2)
-					var/bdrain = rand(1,25)
-					D << "<span class='danger'>You feel weakened.</span>"
-					D.take_overall_damage(bdrain, 0)
-					drain += bdrain
-	if(!drain)
-		return fizzle()
-	usr.say ("Yu[pick("'","`")]gular faras desdae. Havas mithum javara. Umathar uf'kal thenar!")
-	usr.visible_message("<span class='danger'>Blood flows from the rune into [usr]!</span>", \
-	"<span class='danger'>The blood starts flowing from the rune and into your frail mortal body. You feel... empowered.</span>", \
-	"<span class='italics'>You hear a liquid flowing.</span>")
-	var/mob/living/user = usr
-	if(user.bhunger)
-		user.bhunger = max(user.bhunger-2*drain,0)
-	if(drain>=50)
-		user.visible_message("<span class='danger'>[user]'s eyes give off eerie red glow!</span>", \
-		"<span class='danger'>...but it wasn't nearly enough. You crave, crave for more. The hunger consumes you from within.</span>", \
-		"<span class='italics'>You hear a heartbeat.</span>")
-		user.bhunger += drain
-		src = user
-		spawn()
-			for (,user.bhunger>0,user.bhunger--)
-				sleep(50)
-				user.take_overall_damage(3, 0)
+		offering = possible_targets[possible_targets.len]
+	if(!offering)
 		return
-	user.heal_organ_damage(drain%5, 0)
-	drain-=drain%5
-	for (,drain>0,drain-=5)
-		sleep(2)
-		user.heal_organ_damage(5, 0)
-	return
-
-
-///Raise Dead: Requires two corpses. Brings one to life and gibs the other.
-
-
-/obj/effect/rune/proc/raise()
-	var/mob/living/carbon/human/corpse_to_raise
-	var/mob/living/carbon/human/body_to_sacrifice
+	if(offering.null_rod_check())
+		user << "<span class='warning'>Something is blocking the Geometer's magic!</span>"
+		log_game("Sacrifice rune failed - target has null rod")
+		fail_invoke()
+		return
+	if(ishuman(offering) || isrobot(offering))
+		var/cultists_nearby = 1
+		for(var/mob/living/M in orange(1,src))
+			if(iscultist(M) && M != user)
+				cultists_nearby++
+				M.say(invocation)
+		if(cultists_nearby < 3)

```

Comment:
The sacrifice rune requires three cultists if the target is human or robotic, but only one if it's a simple animal or monkey.